src/operations/paintLine.ts

```diff
@@ -0,0 +1,84 @@
+import { IJS } from '../IJS';
+import checkProcessable from '../utils/checkProcessable';
+import { getOutputImage } from '../utils/getOutputImage';
+
+export interface Point {
+  /**
+   * point row
+   *
+   */
+  row: number;
+  /**
+   * point column
+   *
+   */
+  column: number;
+}
+export interface PaintLineOptions {
+  /**
+   * Array of 3 elements (R, G, B),or 4 elements (R, G, B, A) default is red.
+   *
+   * @default [image.maxValue, 0, 0]
+   */
+  color?: [number, number, number] | [number, number, number, number];
+  /**
+   * Image to which the resulting image has to be put.
+   */
+  out?: IJS;
+}
+/**
+ * Paint a polyline defined by an array of points.
```

Comment:
This method doesn't paint a polyline.

Reply:
yes sorry i also implemented polyline i will create the PR after mergin this one